ui/components/multichain/detected-token-banner/detected-token-banner.js

```diff
@@ -23,14 +25,32 @@ export const DetectedTokensBanner = ({
 }) => {
   const t = useI18nContext();
   const trackEvent = useContext(MetaMetricsContext);
+  const { tokenNetworkFilter } = useSelector(getPreferences);
+
+  const allNetworksFilterShown = Object.keys(tokenNetworkFilter ?? {}).length;
 
   const detectedTokens = useSelector(getDetectedTokensInCurrentNetwork);
-  const detectedTokensDetails = detectedTokens.map(
-    ({ address, symbol }) => `${symbol} - ${address}`,
-  );
 
+  const detectedTokensMultichain = useSelector(
+    getAllDetectedTokensForSelectedAddress,
+  );
   const chainId = useSelector(getCurrentChainId);
 
+  const detectedTokensDetails =
+    process.env.PORTFOLIO_VIEW && !allNetworksFilterShown
```

Comment:
Isn't the condition `process.env.PORTFOLIO_VIEW` sufficient here?

Reply:
no because the list of detected token depend also on the drop down menu where the user choose current network or all network on the assets list